nxrm-ha/values.yaml

```diff
@@ -242,6 +241,12 @@ secret:
       file: # Specify the license file name with --set-file license.licenseSecret.file="file_name" helm option
       fileContentsBase64: your_license_file_contents_in_base_64
       mountPath: /var/nexus-repo-license
+  nexusSecret:
+    enabled: false
+    name: nexus-secret.json
+    nexusSecretsKeyId: "super_secret_key_id"
```

Comment:
not an expert on the topic but , does that mean we need to pass a secret key id param too? 

If that's correct , the file itself contains the id of the active key to be used later for encryption